app/safe/internal/bootstrap/privates.go

```diff
@@ -48,6 +65,9 @@ func persistKeys(privateKey, publicKey, aesSeed string) error {
 			ObjectMeta: metaV1.ObjectMeta{
 				Name:      env.SafeAgeKeySecretName(),
 				Namespace: env.SystemNamespace(),
+				Annotations: map[string]string{
+					"kubectl.kubernetes.io/last-applied-configuration": string(secretConfigJSON),
```

Comment:
Not providing this was resulting in a warning when `make deploy` or its variants did a `kubectl apply -f` on that secret.